### PR TITLE
add option to display AvatarData receive stats with display name

### PR DESCRIPTION
**Pull request by birarda**

### CODE REVIEW

**Comment by birarda** [comment]
need to pull a commit out of here that slipped in - one sec

**Comment by hifi-gustavo** [comment]
Build console output for commit: be8a74855882f534c8ae4ad67082a19dc53963c7 available at: https://gist.github.com/82a4b92ebf265e4f18f5<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4722.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4722.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 1995fa6a5ee53fe3d58e8cb8179372334114f811 available at: https://gist.github.com/82a4b92ebf265e4f18f5<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4722.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4722.dmg

File: libraries/avatars/src/AvatarData.h [hunk 1]
```diff
@@ -395,6 +396,8 @@ public slots:
 
     AABox _localAABox;
 
+    SimpleMovingAverage _averageBytesReceived {};
```
Comment (Atlante45) [inline libraries/avatars/src/AvatarData.h]: Just curious, how is that compiling? Don't you need an '=' here?
`SimpleMovingAverage _averageBytesReceived = {}`

Comment (AndrewMeadows) [inline libraries/avatars/src/AvatarData.h]: I think, in this context the open-close brace is just a block of code that us pure no-op so it compiles out fine.  The end result would be... _averageBytesReceived is declared but not explicitly initialized.  There is no special handling for _averageBytesReceived in the AvatarData ctor so it uses the default SimpleMovingAverage ctor.

Comment (birarda) [inline libraries/avatars/src/AvatarData.h]: Yeah, it's not needed at all then, right? I had an argument before to the SimpleMovingAverage constructor and then ended up removing it

**Comment by hifi-gustavo** [comment]
Build console output for commit: 61b4025526e798318f226912f26e5bbff46832fb available at: https://gist.github.com/82a4b92ebf265e4f18f5<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4722.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4722.dmg

**Pull request merged**